### ISSUE TITLE
Quikz Init

### ISSUE THREAD

**Issue by shubhansu-kr** [mvp-2003/Quikz#1]
We need to initialise the project

**Comment by mvp-2003** [comment]
Yes Yes. If you want, you can go ahead and start it. I will merge it. Thank you.

**Comment by shubhansu-kr** [comment]
Sure would start working on this. Please assign me this issue.

**Comment by mvp-2003** [comment]
There you go @shubhansu-kr. All the best.

**Comment by shubhansu-kr** [comment]
### Set up project structure and add standard files

**Tech Stack:**
- MERN Stack (MongoDB, Express, React, Node.js)

**Description:**
We are initializing the repository for Quikz application. The proposed structure will be as follows:

/src
   - /client
   - /server

...other config files

Tasks for this issue include:
1. Create the `/src` directory and two subdirectories: `/client` for the React frontend and `/server` for the Express backend.
2. Add the following standard files to the repository root:
   - `README.md`: Provides an overview of the project and setup instructions.
   - `CONTRIBUTING.md`: Guidelines for contributing to the project.
   - `CODE_OF_CONDUCT.md`: Code of conduct for all contributors.
   - `.gitignore`: Specify files and directories to be excluded from version control.
   - `LICENSE`: Add a suitable open-source license (e.g., MIT License).

This issue ensures the repository is properly structured and prepared for further development.

---

### **Acceptance Criteria:**
- Directory structure is created as described.
- All standard files are added with relevant content.

---

Sub Task created 
1. Client Setup: #2 
2. Server Setup: #3

**Comment by shubhansu-kr** [comment]
@mvp-2003 

Please assign the sub-issues and add appropriate labels (hacktoberfest) to all the issues. I will be creating pull request from 1st of october. 

Please let me know if you want any changes or have any suggestions for the created issue/task.

Thanks

**Comment by mvp-2003** [comment]
I will add Hacktoberfest to all the 3 issues. Ok?

**Comment by shubhansu-kr** [comment]
Thank you @mvp-2003 , I am working on these issue. Please let me know if you have any suggestions.

**Comment by mvp-2003** [comment]
No as of now I think we are good

**Comment by shubhansu-kr** [comment]
Update : 

1. Started working on the issue. 
2. Forked Repo [here](https://github.com/shubhansu-kr/Quikz)
3. Created Branch for feature. Check Branch [here](https://github.com/shubhansu-kr/Quikz/tree/project-init).

Will be creating PR in october.

**Comment by shubhansu-kr** [comment]
Hi @mvp-2003 , I have created 3 branches for 3 issues. However each branch is checked out from the previous one so there is a dependency of each branch on the previous one. 

Sequence : 
1. Checkout branch 1 from main
2. Branch 1 : Commit changes for issue 1. 
3. Checkout branch 2 from branch 1.
4. Branch 2 : Commit changes for issue 2.
5. Checkout branch 3 from branch 2. 
6. Branch 3 : Commit changes for issue 3.

So to create a PR for all 3 i would need to follow the sequence is merging too. Thus i would have to get contributor access to merge in correct sequence without any communication hassle. 

Requesting to get contributor permission for this repository. 

Thanks.
